### DISCUSSION TITLE
Is there a way to generate a .ksplat file with a python script ?

### DISCUSSION THREAD

**Discussion by mathiaswandel** [mkkellogg/GaussianSplats3D#353] [Q&A] [answered]
Hello, 
first of all, thank you for your work, the player is amazing.

I have a question. After training a model, I have my PLY file, and I want to convert it to a KSPLAT file.
But, I want to do it automatically with a Python script. Do you have a Python version of your file `create-ksplat.js` ?

I've tried to do it myself but there is too many conditions with the gaussian-splat-3d module.

**Comment by mkkellogg** [comment, marked as answer]
Right now, all the format conversion code is fairly tightly coupled to the code in the Viewer, so it will definitely take some work to break it out and convert it to Python. Someday, hopefully I will get around to it!
